### PR TITLE
Added 1 new line, on line 132.

### PR DESCRIPTION
**Pull request by Firesphere**
Git thinks I changed the entire template. I've tried many ways to prevent it, but the moment I add just one line on line 132, in whatever editor, Git thinks it's the entire template.

The actual change is line 132, stating that updates aren't real time.

### CODE REVIEW

**Comment by dhensby** [comment]
It's because you're changing whitespace, probably line endings.

File: themes/addons/templates/Layout/Addon.ss [hunk 1]
```diff
@@ -1,254 +1,257 @@
...
+			<p>Module rating system helping users find modules that are well supported. For more on how the rating system works visit <a href="http://www.silverstripe.org/software/addons/supported-modules-definition/">Module standards</a></p>
+			<p><small>Score not correct? <a href="mailto:community@silverstripe.com">Let us know there is a problem</a></small></p>
+            <p><small>Please be aware that the scoring is not updated realtime. It can take several days to update. Don't worry if it's not updated immediately, give us some time.</small></p>
+		</div>
+
+		<% if $HelpfulRobotData %>
+			<% with $HelpfulRobotData %>
+		<div class="span3 offset1">
+			<div class="rating-item">
+				<div class="circle circle-option-sml<% if $has_readme_file %> green<% else %> grey<% end_if %>">
```
Comment (dhensby) [inline themes/addons/templates/Layout/Addon.ss]: This should really just be a loop over all these properties :/

**Comment by Firesphere** [comment]
Even TextEdit changes the line endings? That's stupid.
And how would you loop them? They're all different properties, thus a `with` makes more sense to me.

**Comment by robbieaverill** [comment]
Can we push to an ArrayList which could be looped?

**Comment by dhensby** [comment]
@Firesphere you need to change the branch you're making the PR from, you're using your master branch and it's polluting the commit history...

**Comment by Firesphere** [comment]
@dhensby Sure, which branch should I use? 2.0?

**Comment by dhensby** [comment]
you can target master, but don't use *your* master

**Pull request closed without merging**